doc/source/whatsnew/v1.3.0.rst

```diff
@@ -31,6 +31,37 @@ Notable bug fixes
 
 These are bug fixes that might have notable behavior changes.
 
+``Categorical.unique`` now always maintains same dtype as original
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, when calling :meth:`Categorical.unique`, unused categories in the new array
```

Comment:
Maybe we should use the phrazing "`unique()` with categorical data", since most people will not interact directly with `Categorical.unique()` method, but rather with a categorical Series?